frontend/src/libs/components/meditation-tracks-list/meditation-track/styles.module.scss

```diff
@@ -0,0 +1,81 @@
+.track {
+  position: relative;
+  width: 100%;
+  max-width: 451px;
+  height: 177px;
+  margin: 0 auto;
+  border-radius: 30px;
+  box-shadow: 0 4px 23px 0 var(--violet-100);
+}
+
+.background-image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  width: 37%;
+  height: 100%;
+  border-radius: 30px 0 0 30px;
+}
+
+.content {
+  position: absolute;
+  right: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 70%;
+  max-width: 85%;
+  height: 100%;
+  padding-right: 23px;
+  padding-left: 29px;
+  background-color: var(--white);
+  border-radius: 30px;
+}
+
+.info {
+  width: 52%;
+  height: 85px;
+  line-height: 20px;
```

Comment:
use value like 1.2 for line-height not in pixels

Reply:
fixed